pages/common/qmv.md

```diff
@@ -0,0 +1,24 @@
+# qmv
+
+> Move files and directories with your favorite text editor.
+> More information: <https://www.nongnu.org/renameutils/>.
+
+- Move a single file (invoke an editor with source on the left, target on the right):
+
+`qmv {{source_file}}`
+
+- Move multiple JPG files:
+
+`qmv {{*.jpg}}`
+
+- Move directories:
+
+`qmv -d {{directory/}} {{another_directory/}} {{the_other_directory/}}`
+
+- Move files/directories inside a directory:
+
+`qmv -R {{directory/}}`
```

Comment:
path/to/directory

Reply:
also fix in bb2f8e0566cad4f47c037ca682480b136aa70ded.